ref/feature/JWT/examples.doc

```diff
@@ -0,0 +1,12 @@
+
+**Examples**
+
+To obtain JSON Web Token (JWT) tokens from the JWT token generation endpoint in the jwt-1.0 feature
+
+- Authenticate with your credentials to access the JWT token generation endpoint at the following URL:
+`https://<host>:<port>/jwt/ibm/api/<configId>/token
+
+After authentication succeeds, the MP-JWT format token is returned in JSON form. The token contains the user and groups for the user as recorded in the user registry of the server.
```

Comment:
The token contains the user and groups for the user as recorded in the user registry of the server.

suggested edit

The token contains the user ID and any groups to which the user belongs, as recorded in the user registry of the server.